core/src/scss/utilities/placeholders/components/_cta.scss

```diff
@@ -22,6 +22,7 @@
 
   .su-cta__button {
     @include button-big;
+    @include margin(null null 0);
```

Comment:
Why was this necessary? Just curious.

Reply:
In the current one, when you tab to the CTA, in a couple of them, there's a white space the whole CTA (more obvious when you tab to it so you see the focus outline).

Reply:
Ok, I see why you want to remove the bottom margin on the CTA button. But this has the effect of changing the vertical spacing of the entire CTA. Should we not just move the bottom margin to the whole CTA, vs. eliminating it?

Reply:
I believe we don't actually want any external spacing that's outside any Decanter elements (no spacing outside the top level wrapper) so I'm removing it all together.

Reply:
Hmmm. Yes, I agree with that in principle. However, the fact is we did effectively have external spacing on the CTA component. If people upgrade to the latest master, their layout will be affected if they use CTA's. Maybe that's ok, but I want to make a conscious decision about it.

@sherakama, your thoughts? Are CTA's used on any of your decanter-based sites? Is SWS okay with the bottom margin going away?

Reply:
That's a good point about backward compatibility 😄

Reply:
CSS can change and the styles applied to elements can change, esp, if the new code keeps the end result the same. Keep in mind, we try to avoid top margin and try to only apply margin to the bottom of elements to stay consistent.